src/evtx_parser.rs

```diff
@@ -87,6 +87,8 @@ pub struct ParserSettings {
     num_threads: usize,
     /// If enabled, chunk with bad checksums will be skipped.
     validate_checksums: bool,
+    /// If true, XML attributes will be separated in JSON
+    separate_attributes: bool,
```

Comment:
Please add a brief explanation (simple example)

Reply:
added example and explanation: https://github.com/forensicmatt/evtx/commit/c125ed13ac06d8434a64b74c57511174e7c393ea